Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -1142,6 +1158,276 @@ addDriversInFleet merchantShortId opCity fleetOwnerId req = do
     cleanCSVField idx fieldValue fieldName =
       cleanField fieldValue & fromMaybeM (InvalidRequest $ "Invalid " <> fieldName <> ": " <> show fieldValue <> " at row: " <> show idx)
 
+sendAuthUrlToDriver :: Text -> Id DM.Merchant -> Id DMOC.MerchantOperatingCity -> DP.Person -> Text -> TransporterConfig -> Flow ()
+sendAuthUrlToDriver mobileNumber merchantId merchantOpCityId fleetOwner driverId transporterConfig = do
+  smsCfg <- asks (.smsCfg)
+  let countryCode = fromMaybe "+91" fleetOwner.mobileCountryCode
+      phoneNumber = countryCode <> mobileNumber
+      otp = "" -- required ?
+      authUrl = (fromMaybe "" transporterConfig.bppDashboardUrl) <> "/driver/" <> fleetOwner.id.getId <> "/" <> driverId <> "/fleet/consent/" <> otp
+
+  url <- getShortUrlForFleet authUrl
+  withLogTag "sending Auth Url SMS" $ do
+    (mbSender, message) <-
+      MessageBuilder.buildFleetAuthForDriverMessage merchantOpCityId $
+        MessageBuilder.BuildFleetAuthForDriverMessage
+          { fleetOwnerName = fleetOwner.firstName,
+            fleetAuthUrl = url
+          }
+    let sender = fromMaybe smsCfg.sender mbSender
+    Sms.sendSMS merchantId merchantOpCityId (Sms.SendSMSReq message phoneNumber sender) >>= Sms.checkSmsResult
+
+sendDeepLinkForAuth :: DP.Person -> Text -> Id DM.Merchant -> Id DMOC.MerchantOperatingCity -> DP.Person -> Context.City -> TransporterConfig -> Flow ()
+sendDeepLinkForAuth person mobileNumber merchantId merchantOpCityId fleetOwner opCity transporterConfig = do
+  let countryCode = fromMaybe "+91" person.mobileCountryCode
+  res <-
+    DReg.authWithOtp
+      True
+      DReg.AuthReq
+        { mobileNumber = Just mobileNumber,
+          mobileCountryCode = Just countryCode,
+          merchantId = merchantId.getId,
+          merchantOperatingCity = Just opCity,
+          registrationLat = Nothing,
+          registrationLon = Nothing,
+          name = Just person.firstName,
+          email = person.email,
+          identifierType = Just DP.MOBILENUMBER
+        }
+      Nothing
+      Nothing
+      Nothing
+      Nothing
+
+  let phoneNumber = countryCode <> mobileNumber
+      baseUrl = fromMaybe "https://nammayatri.in/p?vp={#vp#}" transporterConfig.appUrl -- TODO: move to fleeetConfig
+      url = buildAuthUrl [("vp", "wmb_fleet")] baseUrl
+      deepLink = url <> "&authId=" <> res.authId.getId <> "&otp=" <> res.otpCode
```

Comment:
`authId` and `otp` can be removed. Just will send a wmb_fleet_consent deep link and UI will handle calling consent API with `fleetOwnerId` and `driverId`.